build.go

```diff
@@ -179,8 +179,8 @@ func (m buildModule) getBuildListHandler(c *gin.Context) {
 
 		IsInvalid *bool `form:"isInvalid"`
 
-		Status   *string `form:"status"`
-		StatusID *int    `form:"statusId" binding:"excluded_with=Status"`
+		Status   []string `form:"status"`
+		StatusID []int    `form:"statusId" binding:"excluded_with=Status"`
```

Comment:
nit: Maybe exclude `status` if both are present as `statusID` translates more directly to the database values, and requires less computation? Similar to e.g. `environmentMatch` vs `environment`.

Reply:
I would actually argue the other way around. Much better IMO to only expose the string enum variants as they are much easier to work with in the long run and debug.

The computational effort you're talking about is just a `strings.ToLower` and a switch statement. Premature optimization.

I've created an issue on the subject: https://github.com/iver-wharf/wharf-api/issues/152

Reply:
Main argument was that it should drop the one that is less directly equivalent to the database value, if we're changing that then no objections from me 👍🏻 Agree that storing as enum is nicer as well 👍🏻

Reply:
Was actually not suggesting changing the value stored in the database, though that's an option as well. Just that exposing an integer enum through a REST API is more tedious to deal with than a string enum. Same argument applies to the database layer, so maybe we should change it there to. In Go we can quite easily change it to be a string and keep backward compatibility by implementing the [`driver.Valuer`](https://pkg.go.dev/database/sql/driver#Valuer) interface

Reply:
Ah I see.
Yeah I think having it stored as a string enum in the database as well would be good for the same reasons.